.github/workflows/rubocop.yml

```diff
@@ -0,0 +1,27 @@
+name: Rubocop
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  rubocop:
+    name: Rubocop
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        ruby: [
+            2.4
```

Comment:
I used 2.4 here as I think [we don't need to support 2.3](https://github.com/resque/resque-scheduler/pull/721) but happy to change it if not.

Reply:
Any reason we shouldn't use the latest ruby version here instead?

Reply:
I think not. I went with the min supported instead of the max supported.